### PR TITLE
[codemod] markDynamoStrictTest test_module_init

### PR DESCRIPTION
**Pull request by zou3519**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #116656
* #116655
* #116654
* #116653
* #116652
* #116651
* #116650
* #116649
* #116648
* #116647
* #116646
* #116645
* #116644
* #116643
* #116642
* #116641
* #116640
* #116639
* #116638
* #116637
* #116636
* #116635
* #116634
* #116632
* __->__ #116625
* #116624
* #116622
* #116621
* #116619
* #116618

### CODE REVIEW

File: torch/testing/_internal/dynamo_test_failures.py [hunk 1]
```diff
@@ -422,6 +421,156 @@
...
+    "TestModuleInitCPU.test_nn_NLLLoss_cpu_float64",
+    "TestModuleInitCPU.test_quantized_Hardswish_cpu_float32",
+    "TestModuleInitCPU.test_nn_Linear_cpu_float64",
+    "TestModuleInitCPU.test_nn_LazyConvTranspose1d_cpu_float32",
+    "TestModuleInitCPU.test_nn_Conv1d_cpu_float32",
+    "TestModuleInitCPU.test_nn_TransformerDecoder_cpu_float32",
+    "TestModuleInitCPU.test_qat_Linear_cpu_float64",
+    "TestModuleInitCPU.test_quantized_BatchNorm3d_cpu_float32",
+    "TestModuleInitCPU.test_nn_LSTM_cpu_float32",
+    "TestModuleInitCPU.test_qat_Conv1d_cpu_float32",
```
Comment (bdhirsh) [inline torch/testing/_internal/dynamo_test_failures.py:573]: oof

**Review by bdhirsh — APPROVED** [review]

**Comment by huydhn** [comment]
Closing this manually as this has been landed to prevent the bot from merging it twice.  My best guess is that GitHub had a hiccup of sort as closing the PR should be done automatically. Removing merging label and add the merged one are done at the very end, so there is no error from the merging process AFAICT.

**Comment by huydhn** [comment]
This looks a bit suspicious because the lowest commit in the stack was closed correctly https://github.com/pytorch/pytorch/pull/116618.

**Pull request closed without merging**